plugin-attribution/src/test/java/com/mapbox/maps/plugin/attribution/generated/AttributionAttributeParserTest.kt

```diff
@@ -35,7 +35,7 @@ class AttributionAttributeParserTest {
     every { context.obtainStyledAttributes(any(), any(), 0, 0) } returns typedArray
     every { typedArray.getString(any()) } returns "pk.token"
     every { typedArray.getBoolean(any(), any()) } returns true
-    every { typedArray.getInt(any(), any()) } returns 2
+    every { typedArray.getInt(any(), any()) } returns 0
```

Comment:
is it intended to change attribution file here?

Reply:
Yeah, returns 2 would cause an index out-of-range error for tests in location component, so I updated the template and all the generated codes are affected.